app/views/users/_form_fields.html.erb

```diff
@@ -36,6 +36,14 @@
   </p>
 <% end %>
 
+<% if policy(User).flag_2sv? %>
+  <p class="checkbox">
+    <%= f.label :requires_2sv do %>
+      <%= f.check_box :requires_2sv %> Require 2-step verification
```

Comment:
To continue the discussion: I think I would have chosen "require_2sv" or "2sv_required", because we're requiring the user to have it, rather than the user requiring us to allow it. Let me know if that doesn't make sense.